.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
```

Comment:
Please add a README.md. You can follow the same structure as we use in [Bubble Chart](https://github.com/akvelon/PowerBI-Bubble-Chart/blob/master/README.md)

Reply:
Done